.github/actions/install-airbyte-ci/action.yml

```diff
@@ -10,20 +10,28 @@ inputs:
     description: "Path to airbyte-ci source"
```

Comment:
How much do you think we're saving with binary airbyte-ci overall? What if we just default to `pip install` all the time instead? If this removes 1+ page of yaml code, I'd say the simplification of the CI flow might be worth 0.1 seconds on each run?

Reply:
We are saving a significant amount of time by:
* not installing Python
* not downloading deps from pip 

Moreover the Python install github action is rate limited and we often reached the limit last year.

This is why @bnchrch worked on building and releasing an airbyte-ci binary (it also simplifies local install of the tool)

Reply:
How does it work without Python tho!? TIL

Reply:
We use [PyInstaller](https://pyinstaller.org/en/stable/) to produce a binary from the python package.
We produce this binary on each push to a branch changing airbyte-ci.
The `latest` version is the one produced on push to `master`.
The release logic is maintained in this [workflow](https://github.com/airbytehq/airbyte/blob/master/.github/workflows/airbyte-ci-release.yml#L1)
The command to build the binary is [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pyproject.toml#L57)